tensorboard/program_test.py

```diff
@@ -26,37 +26,51 @@
 from tensorboard.plugins.core import core_plugin
 
 
+def fake_asset_provider():
```

Comment:
If we wanted to avoid the boilerplate here we could always make the argument still optional, but if omitted, we would generate a dummy zip file that just contains `index.html` with `Hello TensorBoard` or something.  Probably fine to make it required, but just a thought.

Reply:
I prefer things to be a bit more explicit so I will keep it as is.